### ISSUE TITLE
A required function is missing: glfwGetWindowTitle

### ISSUE THREAD

**Issue by ds000000000001** [flathub/com.pokemmo.PokeMMO#26]
Hi,
I tried to run `flatpak run com.pokemmo.PokeMMO` and get this output:
```
tinyfd_version 3.18.1
tinyfd_response GUI zenity
18:14:16,409 |-INFO in ch.qos.logback.classic.LoggerContext[default] - This is logback-classic version ?
18:14:16,411 |-INFO in ch.qos.logback.classic.util.ContextInitializer@10e92f8f - No custom configurators were discovered as a service.
18:14:16,411 |-INFO in ch.qos.logback.classic.util.ContextInitializer@10e92f8f - Trying to configure with ch.qos.logback.classic.joran.SerializedModelConfigurator
18:14:16,414 |-INFO in ch.qos.logback.classic.util.ContextInitializer@10e92f8f - Constructed configurator of type class ch.qos.logback.classic.joran.SerializedModelConfigurator
18:14:16,419 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.scmo]
18:14:16,420 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback.scmo]
18:14:16,432 |-INFO in ch.qos.logback.classic.util.ContextInitializer@10e92f8f - ch.qos.logback.classic.joran.SerializedModelConfigurator.configure() call lasted 6 milliseconds. ExecutionStatus=INVOKE_NEXT_IF_ANY
18:14:16,432 |-INFO in ch.qos.logback.classic.util.ContextInitializer@10e92f8f - Trying to configure with ch.qos.logback.classic.util.DefaultJoranConfigurator
18:14:16,433 |-INFO in ch.qos.logback.classic.util.ContextInitializer@10e92f8f - Constructed configurator of type class ch.qos.logback.classic.util.DefaultJoranConfigurator
18:14:16,434 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
18:14:16,434 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [jar:file:/app/extra/PokeMMO.exe!/logback.xml]
18:14:16,439 |-INFO in ch.qos.logback.core.joran.spi.ConfigurationWatchList@7ce3cb8e - URL [jar:file:/app/extra/PokeMMO.exe!/logback.xml] is not of type file
18:14:16,571 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - Processing appender named [CONSOLE]
18:14:16,571 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - About to instantiate appender of type [ch.qos.logback.core.ConsoleAppender]
18:14:16,580 |-INFO in ch.qos.logback.core.model.processor.ImplicitModelHandler - Assuming default type [ch.qos.logback.classic.encoder.PatternLayoutEncoder] for [encoder] property
18:14:16,619 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - Processing appender named [CONSOLE_TO_FILE]
18:14:16,619 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - About to instantiate appender of type [ch.qos.logback.core.FileAppender]
18:14:16,626 |-INFO in ch.qos.logback.core.model.processor.ImplicitModelHandler - Assuming default type [ch.qos.logback.classic.encoder.PatternLayoutEncoder] for [encoder] property
18:14:16,629 |-INFO in ch.qos.logback.core.FileAppender[CONSOLE_TO_FILE] - File property is set to [log/console.log]
18:14:16,631 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - Processing appender named [MOD_APPENDER]
18:14:16,631 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - About to instantiate appender of type [ch.qos.logback.core.FileAppender]
18:14:16,631 |-INFO in ch.qos.logback.core.model.processor.ImplicitModelHandler - Assuming default type [ch.qos.logback.classic.encoder.PatternLayoutEncoder] for [encoder] property
18:14:16,632 |-INFO in ch.qos.logback.core.FileAppender[MOD_APPENDER] - File property is set to [log/mods.log]
18:14:16,637 |-INFO in ch.qos.logback.classic.model.processor.LoggerModelHandler - Setting additivity of logger [mod] to false
18:14:16,638 |-INFO in ch.qos.logback.core.model.processor.AppenderRefModelHandler - Attaching appender named [MOD_APPENDER] to Logger[mod]
18:14:16,639 |-INFO in ch.qos.logback.classic.model.processor.RootLoggerModelHandler - Setting level of ROOT logger to DEBUG
18:14:16,640 |-WARN in ch.qos.logback.core.model.processor.ImplicitModelHandler - Ignoring unknown property [priority] in [ch.qos.logback.classic.Logger]
18:14:16,641 |-INFO in ch.qos.logback.core.model.processor.AppenderRefModelHandler - Attaching appender named [CONSOLE] to Logger[ROOT]
18:14:16,641 |-INFO in ch.qos.logback.core.model.processor.AppenderRefModelHandler - Attaching appender named [CONSOLE_TO_FILE] to Logger[ROOT]
18:14:16,641 |-INFO in ch.qos.logback.core.model.processor.DefaultProcessor@78b66d36 - End of configuration.
18:14:16,642 |-INFO in ch.qos.logback.classic.joran.JoranConfigurator@5223e5ee - Registering current configuration as safe fallback point
18:14:16,642 |-INFO in ch.qos.logback.classic.util.ContextInitializer@10e92f8f - ch.qos.logback.classic.util.DefaultJoranConfigurator.configure() call lasted 209 milliseconds. ExecutionStatus=DO_NOT_INVOKE_NEXT_IF_ANY

[INFO 2024-11-11 18:14:16 WIB][main] com.pokeemu.client.Client:-1 - Initialized logger in 7 ms
[INFO 2024-11-11 18:14:16 WIB][main] f.Z1:-1 - Loaded 3 config properties.
Exception in thread "main" java.lang.ExceptionInInitializerError
	at org.lwjgl.glfw.GLFW.glfwPlatformSupported(SourceFile)
	at com.pokeemu.client.Client.main(SourceFile)
Caused by: java.lang.NullPointerException: A required function is missing: glfwGetWindowTitle
	at org.lwjgl.system.APIUtil.requiredFunctionMissing(SourceFile:2)
	at org.lwjgl.system.APIUtil.apiGetFunctionAddress(SourceFile)
	at org.lwjgl.glfw.GLFW$Functions.<clinit>(SourceFile)
	... 2 more
```

**Comment by krasmazov483** [comment]
Can confirm same thing on my PC. Also tested on a laptop that didn't have the game installed before and it throws the same error

**Comment by kyu-n** [comment]
Resolved in #27
